apps/session-manager/src/services/index.ts

```diff
@@ -9,3 +11,13 @@ export {
   TokenService,
   LollipopService,
 };
+
+// Exported Services Dependencies
+
+export type RedisSessionStorageServiceDepencency = {
+  redisSessionStorageService: typeof RedisSessionStorageService;
+} & FnAppRepo.FnAppAPIRepositoryDeps;
+
+export type LollipopServiceDepencency = {
+  lollipopService: typeof LollipopService;
+} & LollipopApi.LollipopApiDeps;
```

Comment:
What is the advantage of using services deps injection? I think that this step semplify the testing avoiding the spyOn mocking, but have an initialization boilerplate in main app.

Reply:
I find it beneficial to define the service layer as a dependency of the controller layer. This approach promotes better separation of concerns and enhances the maintainability and testability of our codebase.

Reply:
Ok, IMHO is suboptimal carry around something that can be simply used

Reply:
Can we rethink about it in next refactor?